### PR TITLE
pacific: rgw: allow to set ssl options and ciphers for beast frontend

### PR DESCRIPTION
**Pull request by k0ste**
backport tracker: https://tracker.ceph.com/issues/51699

---

backport of https://github.com/ceph/ceph/pull/41579
parent tracker: https://tracker.ceph.com/issues/50932

this backport was staged using ceph-backport.sh version 16.0.0.6848
find the latest version at https://github.com/ceph/ceph/blob/master/src/script/ceph-backport.sh

### CODE REVIEW

File: PendingReleaseNotes [hunk 1]
```diff
@@ -9,6 +9,29 @@
...
+  on the file system.
+
+* RGW: `radosgw-admin realm delete` is now renamed to `radosgw-admin realm rm`. This
+  is consistent with the help message.
+
+* OSD: Ceph now uses mclock_scheduler as its default osd_op_queue to provide QoS.
+
+* RGW: S3 bucket notification events now contain an `eTag` key instead of `etag`,
+  and eventName values no longer carry the `s3:` prefix, fixing deviations from
+  the message format observed on AWS.
```
Comment (cbodley) [inline PendingReleaseNotes:28]: these release notes weren't part of the original PR

**Review by cbodley — CHANGES_REQUESTED** [review]

**Comment by yuriw** [comment]
This can be merged as soon as comments resolved

File: PendingReleaseNotes (hunk 1 above)
Comment (k0ste) [reply PendingReleaseNotes:28]: Removed

**Review by cbodley — APPROVED** [review]
thanks!

**Comment by cbodley** [comment]
jenkins test make check

**Comment by yuriw** [comment]
@k0ste pls merge if all is finished here

**Comment by k0ste** [comment]
@yuriw I think you mean @cbodley 🙂

**Pull request merged**